app/views/home/opensource.html.erb

```diff
@@ -1,5 +1,91 @@
-<section class="opensource-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto">
+<section class="opensource-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto border-bottom">
   <div class="container cointaier-shrinked">
-    <h1 class="display-4 text-left">OpenSource</h1>
+    <h1 class="display-4 text-left text-success">OpenSource Projects</h1>
+    <h2 class="text-muted text-left">We offer free hosting to selected open source projects.</h2>
   </div>
-</section>
\ No newline at end of file
+</section>
+<section class="pt-8 pt-md-11 bg-light p-0 p-md-5">
+  <div class="container carousel-opensource d-none d-md-block"> 
```

Comment:
it's possible to have no featured project (@featured_projects = [])

Reply:
Done.